src/library_memfs.js

```diff
@@ -274,6 +274,9 @@ mergeInto(LibraryManager.library, {
       },
 
       // Writes the byte range (buffer[offset], buffer[offset+length]) to offset 'position' into the file pointed by 'stream'
+      // canOwn: A boolean that tells if this function can take ownership of the passed in buffer from the subbuffer portion
+      //         that the typed array view 'buffer' points to. The underlying ArrayBuffer can be larger than that, but
+      //         canOwn=true will not take ownership of the portion outside the bytes addressed by the view.
       write: function(stream, buffer, offset, length, position, canOwn) {
         if (!length) return 0;
         var node = stream.node;
```

Comment:
a few lines below this (github won't let me comment there...) is a comment "Can we just reuse the buffer we are given?", let's update that. It can really just be deleted.